modules/ols-creating-lightspeed-custom-resource-file-using-web-console.adoc

```diff
@@ -18,7 +18,7 @@ The Custom Resource (CR) file contains information that the Operator uses to dep
 
 .Procedure 
 
-. Click *Add* in the upper-right corner of the {ocp-short-name} web console.
+. Click the *Quick create* icon in the upper-right corner of the {ocp-short-name} web console and select *Import YAML*.
```

Comment:
In the [web console tutorials](https://github.com/openshift/openshift-docs/blame/main/modules/getting-started-web-console-connecting-a-database.adoc#L18), it appears that we use _the *Quick create* menu_. Should we be consistent here?
```suggestion
. Click the *Quick create* (image:fa-plus-circle.png[title="Quick create menu"]) menu in the upper-right corner of the {ocp-short-name} web console and select *Import YAML*.
```

Reply:
done